pyquil/tests/data/qmi_auth_token_valid.json

```diff
@@ -0,0 +1,4 @@
+{
+  "access_token": "ok",
+  "refresh_token": "supersecret"
+}
```

Comment:
put this and all the other new data files in a `data` folder in the `api/tests` dir (will have to change ur `fixture_dir` file)

Reply:
so the tests will live in the top level but their fixtures will live in a subdir?